app/models/questionable/answer.rb

```diff
@@ -1,8 +1,10 @@
 module Questionable
   class Answer < ActiveRecord::Base
     belongs_to :user
-    belongs_to :assignment
     belongs_to :option
+    belongs_to :parent_question, class_name: 'Questionable::Question', foreign_key: 'question_id'
+
+    belongs_to :assignment
     has_one :question, through: :assignment
```

Comment:
I was going to remove

```
belongs_to :assignment
has_one :question, through: :assignment
```

after step 2 given that the new association is named `parent_question` and until step 2 there's gonna be code on bespoke that uses `question` instead of `parent_question`.